box/aiModels.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "model_id": "noop",
+    "pipeline": "live-video-to-video",
+    "warm": true
+  },
+  {
+    "model_id": "noop",
+    "pipeline": "live-video-to-video",
+    "warm": true
+  },
```

Comment:
Shouldn't this be coupled with a change somwhere setting `-aiRunnerContainersPerGPU=3` for the box O?

Reply:
Ahh, right, let me revert it for now.